### PR TITLE
Add bash lsp.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Add bash lsp. Should be useful for dog fooding.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: v3/plugins/bash-lsp/bash-ide/1.10.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+name: bash-ide
+version: 1.10.2
+type: VS Code extension
+displayName: Bash IDE
+title: A launguage server for Bash.
+description: Visual Studio Code extension utilizing the bash language server, that is based on Tree Sitter and its grammar for Bash and supports explainshell integration.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/bash-lsp/bash-language-server
+category: Other
+firstPublicationDate: '2020-07-4'
```
Comment (ericwill) [inline v3/plugins/bash-lsp/bash-ide/1.10.2/meta.yaml:12]: Same comment as the other PR, please add a 0 to the day, so '2020-07-04'

**Comment by ericwill** [comment]
Also, the `latest.txt` file is missing.

**Comment by benoitf** [comment]
```
!!! Location mismatch in plugin 'Mads Hartmann/bash-ide/latest':
!!!   Expected location: 'v3/plugins/Mads Hartmann/bash-ide/latest/meta.yaml'
!!!   Actual location: 'v3/plugins/bash-lsp/bash-ide/latest/meta.yaml' 
```

File: vscode-extensions.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "extensions": [
+    {
+      "repository": "https://github.com/bash-lsp/bash-language-server",
+      "revision": "vscode-client-1.10.2"
```
Comment (benoitf) [inline vscode-extensions.json:5]: I think it's missing 
```json
"directory": "vscode-client"
```

File: vscode-extensions.json [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+    },
```
Comment (benoitf) [inline vscode-extensions.json:6]:
```suggestion
      "revision": "vscode-client-1.10.2",
      "directory": "vscode-client"
    },
```

**Comment by benoitf** [comment]
let's squash and merge 🎉

**Pull request merged**